odl/solvers/iterative/iterative.py

```diff
@@ -65,6 +62,9 @@ def landweber(op, x, rhs, niter=1, omega=1, partial=None):
     convergence, where :math:`\\lVert\mathcal{A}\\rVert` stands for the
     operator norm of :math:`\mathcal{A}`.
 
+    Users may also optionally provide a projection operator to project each
```

Comment:
I wouldn't write operator here since users may think they have to use `Operator`.